OpenEdXMobile/src/androidTest/java/org/edx/mobile/test/feature/LaunchFeatureTest.java

```diff
@@ -27,13 +29,12 @@ public void whenAppLaunched_withValidUser_myCoursesScreenIsShown() throws Except
     }
 
     @Test
-    public void whenAppLaunched_withInvalidAuthToken_logInScreenIsShown() {
+    public void whenAppLaunched_withInvalidProfile_landingScreenIsShown() {
         environment.getLoginPrefs().storeAuthTokenResponse(TestValues.INVALID_AUTH_TOKEN_RESPONSE, LoginPrefs.AuthBackend.PASSWORD);
-        environment.getLoginPrefs().storeUserProfile(TestValues.DUMMY_PROFILE);
+        environment.getLoginPrefs().storeUserProfile(null); // Make sure that profile is null so we are going back to landing screen
         new AppInteractor()
                 .launchApp()
-                .observeLogInScreen()
-                .navigateBack()
-                .observeLandingScreen();
+                .observeLandingScreen(); // If profile is not set &
```

Comment:
Was wondering why the logic here changes, and then I realized we don't run these on CI and they are a bit outdated:(